circuit/src/scaffold.ts

```diff
@@ -23,7 +20,7 @@ export abstract class AxiomBaseCircuitScaffold<T> extends BaseCircuitScaffold {
   protected halo2Lib!: Halo2LibWasm;
   protected provider: string;
   protected dataQuery: DataSubquery[];
```

Comment:
there is already dataQuery in queryBuilder: can it be re-used? otherwise the redundancy is going to be error prone.

Reply:
was able to remove queryBuilderBase from here